Add documenation and make sure runs without options. (#17)
redhat-performance/fio-wrapper: README.md

Automation wrapper for fio

Description:
                Fio spawns a number of threads or processes doing a particular type
                of I/O action as specified by the user. fio takes a number of global

>                 parameters, each inherited by the thread unless otherwise parameters
>                 given to them overriding that setting is given. The typical use of

Reviewer: ```suggestion
                parameters, each inherited by the thread unless other parameters
                provided for that thread overrides that global setting. The typical use of
```

Cleanup grammar

                fio is to write a job file matching the I/O load one wants to simulate.

Location of underlying workload: https://fio.readthedocs.io/en/latest/fio_doc.html
   benchmark loaded via dnf

Packages required: gcc,numactl,python3,bc,fio

To run:
```
[root@hawkeye ~]# git clone https://github.com/redhat-performance/fio-wrapper
[root@hawkeye ~]# fio-wrapper/fio/fio_run
```

Defaults for the script are the following:
```
	devices: No default, if not provided will be prompted
        run time: 120 seconds
        test type: read and write
        ioengine: libaio
        jobs min: 1
        jobs max: # numa nodes
        block size: 4k and 1024k
	io depth: 1 2 4 8 16 32
       
If regression option is selected, only one set of runs with all the disks.
	devices: No default, if not provided will be prompted
        run time: 120 seconds
        test type: read and write
        ioengine: libaio
        jobs min: 1
        jobs max: # numa nodes
        block size: 4k and 1024k
	io depth: 1 16 64
```
Options

> Usage: fio-wrapper/fio/fio_run

Reviewer: ````suggestion
```
Usage: fio-wrapper/fio/fio_run
````
